lnrpc/routerrpc/router.proto

```diff
@@ -354,6 +367,67 @@ message PairData {
     int64 success_amt_msat = 7;
 }
 
+message GetMissionControlConfigRequest {
+}
+
+message GetMissionControlConfigResponse {
+    /*
+    Mission control's currently active config.
+    */
+    MissionControlConfig config = 1;
+}
+
+message SetMissionControlConfigRequest {
+    /*
+    The config to set for mission control. Note that all values *must* be set,
+    because the full config will be applied.
+    */
+    MissionControlConfig config = 1;
+}
+
+message SetMissionControlConfigResponse {
+}
+
+message MissionControlConfig {
+    /*
+    The amount of time mission control will take to restore a penalized node
+    or channel back to 50% success probability, expressed as a unix timestamp
+    in seconds. Setting this value to a higher value will penalize failures for
+    longer, making mission control less likely to route through nodes and
+    channels that we have previously recorded failures for.
+    */
+    uint64 half_life_seconds = 1;
+
+    /*
+    The percentage probability of success mission control should assign to hop
+    in a route where it has no other information available. Higher values will
+    make mission control more willing to try hops that we have no information
+    about, lower values will discourage trying these hops.
+    */
+    uint32 hop_probability = 2;
```

Comment:
I'm a bit unsure about using integer percents for the probability. Maybe better to use floats since it results in simpler code and also less confusing? Also for weight?